README.md

```diff
@@ -1 +1,60 @@
 # Core Domain Charts
+
+Core Domain Charts help you to visualise the strategic importance of each sub-domain or business capability in your software architecture allowing you to make business model-aligned architectural decisions.
+
+[Core Domains]() are the parts of your domain where the expected ROI curve is greatest, and deserve the highest focus.
```

Comment:
I would add a link to an explanation to `ROI` to make it more beginners friendly

Reply:
Thanks. Link added.